### ISSUE TITLE
responds to wrong message

### ISSUE THREAD

**Issue by Geczy** [scmanjarrez/EdgeGPT-Telegram-Bot#48]
![image](https://user-images.githubusercontent.com/1036968/232889423-f6343c84-a1cf-4f7e-97e4-de8960cc4c24.png)

i wasn't @ing bing but it responded anyway

**Comment by scmanjarrez** [comment]
Does the first bot have privacy mode disabled?

**Comment by Geczy** [comment]
not sure what that is, the first bot is an admin, bing bot is not an admin, if that helps

**Comment by scmanjarrez** [comment]
Yeah, that's the problem. If the bot is admin, it read every message sent to the group. Remove admin privileges (it doesn't need them) and it should work as expected (only interact with messages replying to bot or commands)
